src/pages/Members/Members.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import './Members.scss';
+
+import girl_holding_books from '../../assets/girl_holding_books.png';
+
+export const Members: React.FC = () => (
+  <div className="members-section">
+    <div className="members-section-left">
+      <h2 className="heading2-underlined">our great members</h2>
+      <span />
+      <p className="members-description">
+        Our community is made up of book lovers from all walks of life. We have
+        students, teachers, parents, and professionals who are passionate about
+        reading and sharing their thoughts with others. Join us today and meet
+        new friends who share your love for reading.
+      </p>
+    </div>
+    <div className="members-section-right">
+      <img
+        className="members-illustration"
+        src={girl_holding_books}
+        alt="girl_holding_books"
```

Comment:
That is not very useful alt, isnt it. It should describe to a reader what the imagine is for. What it does, or show. It should be human readable. -> https://www.w3.org/WAI/tutorials/images/decision-tree/

Btw this image is pure decoration, it serves no purpose, we should go with empty alt `alt=""`